e2e/testdata/porch/rpkg-update/config.yaml

```diff
@@ -61,24 +61,3 @@ commands:
       git-f1fbc2b72ee70b2f34e64c2630031d0560e6cf48   basens-edit         v0         false    Draft       git
       git-894137a40c0792e88a9b844491a95a466b085ca1   basens-edit         v2         true     Published   git
       git-542e4d9fd8ca2ee94edf43aadb3ec31dc2fbd3a0   basens-edit-clone   v0         false    Draft       git
-  - args:
-      - alpha
-      - rpkg
-      - update
-      - --namespace=rpkg-update
-      - --discover
-      - git-542e4d9fd8ca2ee94edf43aadb3ec31dc2fbd3a0
```

Comment:
@mortent The current CLI seems incompatible with the changes I made to the backend (see the test failures in https://github.com/natasha41575/kpt/runs/7224748986?check_suite_focus=true).

This removal is temporary, I add it back along with the CLI change in https://github.com/GoogleContainerTools/kpt/pull/3346, which updates the CLI to be compatible with the new backend.